products/extensions/b2b-components/order-approval/04-add-new-approval-condition.md

```diff
@@ -0,0 +1,83 @@
+---
+nav:
+  title: How to add a new approval condition
+  position: 40
+
+---
+
+# How to add a new approval condition
+
+The order approval component provides a set of conditions that you can use to define your approval rules. However, if you need to add a new condition, you can do so via app or plugin.
+
+## Plugin
```

Comment:
could you provide an information that:
# App 
We have not yet added support for extending custom OrderApprovalRules for the app.